recipe/meta.yaml

```
@@ -1,53 +1,81 @@
-{% set version = "0.23.8" %}
+{% set name = "awscrt" %}
+{% set version = "0.27.4" %}
 
 package:
-  name: awscrt
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
-  sha256: cba55f3ee80ea3192a0a24e84caad778570250800a59d29ef9efbcd4d1612f2f
+  url: https://pypi.org/packages/source/a/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb23cb767ca2e3c0007c899b289d668d28aeb055534df35a741847135f6cd91c
   patches:
     - patches/0001-Don-t-force-static-linkage.patch
+    - patches/0002-disable-osx-failing-tests.patch  # [osx]
+    - patches/0003-disable-win-failing-test.patch  # [win]
 
 build:
   number: 0
+  skip: true  # [py<38]
+  script_env:
+    # Unvendor AWS libs
+    - AWS_CRT_BUILD_USE_SYSTEM_LIBS=1
+    - AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=1
+  # These are also in requirements: https://github.com/awslabs/aws-crt-python/blob/v0.27.4/setup.py#L220-L230
+  ignore_run_exports:
+    - aws-c-sdkutils
+    - aws-c-compression
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
-    - patch     # [not win]
+    - patch  # [not win]
     - m2-patch  # [win]
   host:
     - python
     - pip
-    - setuptools
-    - wheel
-    - s2n 1.5.14  # [linux]
-    - aws-c-common 0.11.1
-    - aws-c-sdkutils 0.2.3
-    - aws-c-cal 0.8.5
-    - aws-c-io 0.17.0
-    - aws-checksums 0.2.3
+    - setuptools >=75.3.1
+    - wheel >=0.45.1
+    - s2n {{ s2n }}  # [linux]
+    - aws-c-common 0.12.3
+    - aws-c-sdkutils 0.2.4
+    - aws-c-cal 0.9.2
+    - aws-c-io 0.20.1
+    - aws-checksums 0.2.7
     - aws-c-compression 0.3.1
-    - aws-c-event-stream 0.5.4
-    - aws-c-http 0.9.3
-    - aws-c-auth 0.8.5
-    - aws-c-mqtt 0.12.2
-    - aws-c-s3 0.7.11
+    - aws-c-event-stream 0.5.5
+    - aws-c-http 0.10.2
+    - aws-c-auth 0.9.0
+    - aws-c-mqtt 0.13.2
+    - aws-c-s3 0.8.3
```

https://github.com/awslabs/aws-crt-python/tree/v0.27.4/crt
```suggestion
    - aws-c-s3 0.8.3
    - aws-ls (need to add)
```

---

Maybe I'm wrong, but I tried to configure the building to use the system's encryption lib, such as OpenSSL:
https://github.com/awslabs/aws-crt-python/blob/5cf5e7d70f5ecd3a88fae15fa3f427b82f30fbae/setup.py#L193-L194

From what I understand, this means we don't need aws-lc:
https://github.com/awslabs/aws-crt-python/blob/5cf5e7d70f5ecd3a88fae15fa3f427b82f30fbae/setup.py#L210

In case of Linux, aws-lc is used as a backend for s2n, but I’ve unvendored s2n, and our version uses OpenSSL instead:
https://github.com/awslabs/aws-crt-python/blob/v0.27.4/crt/CMakeLists.txt#L64
https://github.com/AnacondaRecipes/s2n-feedstock/blob/master/recipe/meta.yaml#L35

So in general, since s2n is using OpenSSL, I decided not to mix aws-lc with OpenSSL across different platforms.